src/app/components/ThemeProvider/gridWidths.ts

```diff
@@ -0,0 +1,13 @@
+import { GridWidths } from '#app/models/types/theming';
+
+const gridWidths: GridWidths = {
+  240: 240,
+  360: 360,
```

Comment:
GEL page goes from 0-399, but I think we use these increments somewhere: https://github.com/bbc/simorgh/blob/06bec2372565c9808ddfa8d2cd08fc808e0fcbc1/a11y.js#L9-L10